config/database.yml

```diff
@@ -29,7 +30,6 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user running Rails.
-  #username: music_coop
```

Comment:
It would be good to exclude this change which seems specific to @lenikadali's development environment.

Reply:
This is now resolved